templates/github/pull-request/filters.html.twig

```diff
@@ -107,6 +110,14 @@
                     }
                 });
             });
+
+            function hideEmpty() {
+                $('.empty').map((key, value) => {
+                    let toHide = value.parentElement.parentElement.parentElement.parentElement;
```

Comment:
```suggestion
                    let toHide = value.parentElement.parentElement.parentElement.parentElement;

```

And I use jQuery, so you can use it instead of vanilla.